### PR TITLE
WIP: Mujoco interface through mujoco_py

### PR DESCRIPTION
**Pull request by studywolf**
Basic interface and example script set up

- need to resolve the forward kinematics calculations (set up differently than VREP sim)
- ideally reading from ur5.xml file to get the transformation matrices

### CODE REVIEW

**Comment by p3jawors** [comment]
Rebased to `dynadapt_sampling_fix` branch

**Comment by p3jawors** [comment]
TODO
- remove mujoco import, fix mock imports in dewolf

File: abr_control/arms/jaco2/config.py [hunk 1]
```diff
@@ -55,6 +55,10 @@ def __init__(self, hand_attached=False, **kwargs):
 
         self.REST_ANGLES = np.array([None, 2.42, 2.42, 0.0, 0.0, 0.0])
 
+        # position to move to before switching to torque mode
+        self.START_ANGLES = np.array(
```
Comment (studywolf) [inline abr_control/arms/jaco2/config.py]: why are we adding starting angles?

Comment (p3jawors) [reply abr_control/arms/jaco2/config.py]: I thought we agreed to add this to each arm? I need it for setting a start position

**Comment by studywolf** [comment]
working my way through:
- the examples in the Mujoco folder should be converted to look like the PyGame and VREP examples, i.e. position_controller_osc.py etc
- what's going on with the UR5.xml? should we clean that up or take that out?

**Comment by p3jawors** [comment]
- Removed ur5 mujoco files from git history
- Renamed examples to more closely follow the VREP and Pygame examples
- Combined 1, 2, 3 joint examples into one script `geometric_arm.py` where you can append a 1, 2, or 3 to the function call to specify which arm to use

File: abr_control/arms/jaco2/config.py (hunk 1 above)
Comment (studywolf) [reply abr_control/arms/jaco2/config.py]: hmm, right. does VREP use these start angles too?

Comment (studywolf) [reply abr_control/arms/jaco2/config.py]: also, can you add start angles to the one, two, and three joint configs too?

**Comment by studywolf** [comment]
Pulling in the same set of VREP examples and making sure those all work and I think this will be good for a final review and merge!

**Comment by studywolf** [comment]
everything LGTM, will rebase now

**Pull request merged**